content/docs/getting-started/setup.md

```diff
@@ -63,13 +63,12 @@ Start with the following `cf target` command:
 cf target -o <ORG> -s <SPACE>
 ```
 
-Your `<ORG>` is a Cloud Foundry _organization_ named "sandbox-&lt;agencypart&gt;", where &lt;agencypart&gt; is whatever comes right before `.gov` or `.mil` in your
-e-mail address. For example, this may be `sandbox-gsa` or `sandbox-epa`. In most cases, your `<SPACE>` is the part of your email before the `@`, e.g. `firstname.lastname`. Cloud Foundry _spaces_ let applications run independently within an organization.  
+Your `<ORG>` is a Cloud Foundry _organization_ named "sandbox". In most cases, your `<SPACE>` is your email address. Cloud Foundry _spaces_ let applications run independently within an organization.
```

Comment:
Updated. As an atlas person, I prefer saying "cloud foundry" over "cloud.gov" for concepts that aren't specific to cloud.gov, but agreed that "cloud.gov" will be easier for new users to understand.